packages/python/patch-no-sys-resource_h

```diff
@@ -0,0 +1,29 @@
+--- Modules/_lsprof.c
```

Comment:
It may be possible to remove this patch nowadays. We recently gained a copy of `<sys/resource.h>` that includes a subset of the features of POSIX.